packages/next/lib/caching/valkey-common.js

```diff
@@ -0,0 +1,86 @@
+import { buildPinoFormatters, buildPinoTimestamp } from '@platformatic/foundation'
+import { Redis } from 'iovalkey'
+import { pack, unpack } from 'msgpackr'
+import { existsSync, readFileSync } from 'node:fs'
+import { hostname } from 'node:os'
+import { resolve } from 'node:path'
+import { fileURLToPath } from 'node:url'
+import { pino } from 'pino'
+
+const clients = new Map()
```

Comment:
Global state and global connections are not really observable. Maybe we add this to our capability or globalThis.platformatic?

Reply:
We can't access capability from here. I added to `globalThis.platformatic.valkeyClients`, since we might want to reuse it in the future.